projects/packages/sync/src/modules/class-posts.php

```diff
@@ -138,6 +138,7 @@ public function init_listeners( $callable ) {
 		add_action( 'wp_insert_post', array( $this, 'wp_insert_post' ), 11, 3 );
 		add_action( 'wp_after_insert_post', array( $this, 'wp_after_insert_post' ), 11, 2 );
 		add_action( 'jetpack_sync_save_post', $callable, 10, 4 );
+		add_action( 'save_post_page', $callable, 10, 2 );
```

Comment:
The signature is `do_action( "save_post_{$post->post_type}", $post_ID, $post, $update );` so maybe we do want a comment here to say "We're not going to pass the update itself around unless we need it"

Reply:
Good catch I changed it to be three arguments because I saw that we actually are subscribing to the 3rd argument in wpcom (although not actually using it yet).